style.css

```diff
@@ -20,6 +21,22 @@ a {
   text-decoration: none;
 }
 
+#contents > a > section {
```

Comment:
Because I thought it was best to have a default colour, and then specific ones for sections 2-5…  :)